src/components/snackbar/snackbar.scss

```diff
@@ -9,44 +9,46 @@
     box-sizing: border-box;
 }
 
-.mdc-snackbar {
-    top: var(--snackbar-top, auto);
-    right: var(--snackbar-right, 0);
-    bottom: var(--snackbar-bottom, 0);
-    left: var(--snackbar-left, 0);
```

Comment:
nobody has used these except ourselves in the webclient.

Reply:
Well… it _is_ an open source component library, so there may be people outside the company that has.

But, it won't lead to build errors if they set these variables. It will simply not have any effect. Instead, we provide this nice new functionality where multiple snackbars can be shown at once. I think that's acceptable to do without a major version bump.